src/hive_mind_bridge/src/main.cpp

```diff
@@ -1,5 +1,5 @@
-#include "hive_mind_bridge/Callback.h"
-#include "hive_mind_bridge/HiveMindBridge.h"
+#include "HiveMindBridge/Callback.h"
+#include "HiveMindBridge/HiveMindBridge.h"
```

Comment:
Our libraries usually have snake case for paths, is it possible to change it in the bridge?

Reply:
I can do that.